.github/workflows/sonarcloud.yml

```
@@ -1,86 +1,70 @@
 name: SonarCloud Analysis
+
 on:
   push:
     branches:
-      - main
+      - main # Analyze commits pushed to main
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)